src/overdrive.js

```diff
@@ -4,6 +4,19 @@ import ReactDOM  from 'react-dom'
 const renderSubtreeIntoContainer = ReactDOM.unstable_renderSubtreeIntoContainer;
 const components = {};
 
+const prefix = styles => {
+    const propertiesToPrefix = ['transform', 'transformOrigin', 'transition'];
+    const prefix = 'Webkit';
+    const prefixedStyles = {};
+    Object.keys(styles).forEach(property => {
+        if (propertiesToPrefix.indexOf(property) >= 0) {
```

Comment:
Sorry, just saw this. That looks a little weird to me to use an object for a list. I can move the constants outside the function real quick. Just pushing fixup commits. I can squash once you are happy with them. Also assuming I need to run `npm run build` with this? Do you want that in the same commit for the PR or do you do that as part of your publish process?